src/api/posts/test/posts.test.js

```diff
@@ -11,7 +11,21 @@ describe('/posts', () => {
   const createdItems = 150;
   const nonInteger = 'test';
 
-  const posts = [...Array(createdItems).keys()].map((item) => {
+  const feeds = [...Array(createdItems).keys()].map((item) => {
+    const guid = `http://telescope${item}.cdot.systems`;
+    const id = hash(guid);
+    return {
+      id,
+      author: 'author',
+      url: 'http://foo.url',
+      user: 'user',
+      link: 'http://foo.link',
+      etag: 'etag',
+      lastModified: new Date('2009-09-07T22:23:00.544Z'),
+      githubUsername: 'githubusr',
```

Comment:
s/githubusr/githubUsername/ to make it even more clear.

Reply:
Sorry, what does this mean? Should I change the field name of `githubUsername` to something else? Or the value `githubusr`?

Reply:
The value.  It's easiest if the property name and value match (e.g., like `etag` above).

Reply:
Got it, thank you for the suggestion, it's already fixed now and pushed.